samples/highcharts/blog/blog-update-options/demo.js

```diff
@@ -0,0 +1,787 @@
+Highcharts.theme = {
+    colors: ['#058DC7', '#50B432', '#ED561B', '#DDDF00', '#24CBE5', '#64E572',
+        '#FF9655', '#FFF263', '#6AF9C4'],
+    chart: {
+        backgroundColor: 'transparent'
+    },
+    title: {
+        style: {
+            color: '#000',
+            font: 'bold 16px "Arial"'
+        }
+    },
+    subtitle: {
+        style: {
+            color: '#666666',
+            font: 'bold 12px "Arial"'
+        }
+    },
+    legend: {
+        itemStyle: {
+            font: '9pt Arial"',
+            color: 'black'
+        },
+        itemHoverStyle: {
+            color: 'gray'
+        }
+    }
+};
+// Apply the theme
+Highcharts.setOptions(Highcharts.theme);
+
+Math.easeOutQuint = function (pos) {
+    return (Math.pow((pos - 1), 5) + 1);
+};
+
+let chartMargin = 0;
+let chartSpacing = 0;
+const chartData = [10, 20, 40, 5, 10, 15];
+let axisVisible = false;
+
+///options for each series
+const itemOptions = {
+    marker: {
+        radius: 8
+    },
+    startAngle: -100,
+    endAngle: 100,
+    innerSize: '30%',
+    center: ['50%', '70%'],
+    size: '100%',
+    dataLabels: {
+        enabled: false
+    }
+
+};
+
+const pieOptions = {
+    startAngle: 100,
+    innerSize: '30%',
+    center: ['50%', '60%'],
+    size: '80%',
+    dataLabels: {
+        enabled: false
+    }
+
+};
+const lineOptions = {
+    lineWidth: 10,
+    marker: {
+        radius: 16
+    }
+};
+const columnOptions = {
+    pointWidth: 20,
+    borderRadius: 5,
+    borderWidth: 0,
+    allowOverlap: true,
+    dragDrop: {
+        draggableY: false,
+        draggableX: false,
+        dragHandle: {
+            lineWidth: 0
+        }
+    }
```

Comment:
_Probably_ not needed as we are not loading the `drag-drop` module on this page?